File: VB/10_Conversion/ToHTML.vb
Imports Spire.Presentation

Namespace ToHTML
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create an instance of presentation document
			Dim ppt As New Presentation()

			'Load file
			ppt.LoadFromFile("..\..\..\..\..\..\Data\Conversion.pptx")

			'Save the document to HTML format
			Dim result As String = "ToHTML.html"
			ppt.SaveToFile(result, FileFormat.Html)
			PresentationDocViewer(result)
		End Sub

	Private Sub PresentationDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
	End Sub
	End Class
End Namespace

File: CS/10_Conversion/ToHTML.cs
using System;
using System.Windows.Forms;
using Spire.Presentation;

namespace ToHTML
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create an instance of presentation document
            Presentation ppt = new Presentation();

            //Load file
            ppt.LoadFromFile(@"..\..\..\..\..\..\Data\Conversion.pptx");

            //Save the document to HTML format
            string result = "ToHTML.html";
            ppt.SaveToFile(result, FileFormat.Html);
            PresentationDocViewer(result);
	}
	
	private void PresentationDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}